src/components/header/style.module.css

```diff
@@ -313,6 +307,8 @@
 	justify-content: space-between;
 	align-items: flex-start;
 
+	touch-action: none;
+
```

Comment:
When the header is opened, we should not let the user scroll the page in the background.